src/Sulu/Bundle/ContactBundle/Contact/CustomerManager.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContactBundle\Contact;
+
+use Doctrine\ORM\EntityManager;
+use Sulu\Bundle\ContactBundle\Util\IdsHandlingTrait;
+
+/**
+ * Implements functionality for the manager of account and contact combination.
+ */
+class CustomerManager implements CustomerManagerInterface
```

Comment:
@chirimoya We couldn't think of any better name for the superset of contact and account (the only thing we came up with was to call the base class contact, and have then people and organizations as sub classes).